modules/worker-utils/src/lib/process-utils/child-process-proxy.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-console */
-import {spawn, ChildProcess, SpawnOptionsWithoutStdio} from 'child_process';
+import ChildProcess from 'child_process';
```

Comment:
To avoid naming conflicts, I used the `import * as` syntax and then accessed types from the module object:

```typescript
import * as ChildProcess from 'child_process';

export type ChildProcessProxyProps = {
  /* ... */
  spawn?: ChildProcess.SpawnOptionsWithoutStdio;
  /* ... */
};

export default class ChildProcessProxy {
  /* ... */
  private childProcess: ChildProcess.ChildProcess | null = null;
  /* ... */
}
```